package.json

```diff
@@ -86,6 +84,8 @@
     "nodemon": "1.11.0",
     "nyc": "10.2.0",
     "raml-to-markdown": "^1.0.5",
-    "raml2html": "3.0.1"
+    "raml2html": "3.0.1",
+    "react-router": "^4.0.0",
```

Comment:
At the moment there are no routes in any components. Routes are used just in the dev UI